### PR TITLE
driver-picoquant: add MD_TIME_LIST metadata

### PR DESCRIPTION
**Pull request by philipwinklerdelmic**

### CODE REVIEW

File: src/odemis/driver/picoquant.py [hunk 1]
```diff
@@ -681,6 +681,7 @@ def _acquire(self):
                     md = self._metadata.copy()
                     md[model.MD_ACQ_DATE] = tstart
                     md[model.MD_DWELL_TIME] = tacq
+                    md[model.MD_TIME_LIST] = numpy.linspace(0, self.pixelDuration, self._shape[-1])
```
Comment (pieleric) [inline src/odemis/driver/picoquant.py]: It should be _in place_ of the `MD_PIXEL_DUR` + `MD_TIME_OFFSET`. So, at least, you should remove these two old metadata, and use the time offset also in MD_TIME_LIST.
Moreover, you might want to update the metadata whenever one of the VA changes (so including the binning, now). It's a few extra line, but the advantage is that it's possible to see the metadata from the command line, with `odemis-cli --list-prop time-correlator`, which is handy in times of debugging.

Comment (philipwinklerdelmic) [reply src/odemis/driver/picoquant.py]: just pushed a change

File: src/odemis/driver/picoquant.py [hunk 2]
```diff
@@ -297,6 +296,8 @@ def __init__(self, name, role, device=None, children=None, daemon=None,
 
         # Make sure the device is synchronised and metadata is updated
         self._setSyncOffset(self.syncOffset.value)
+        self._metadata[model.MD_TIME_LIST] = (numpy.linspace(0, self.pixelDuration.value, self._shape[-1]) + \
```
[the remark this answers is not in the document]
Comment (philipwinklerdelmic) [reply src/odemis/driver/picoquant.py]: done!

I also pushed changes for the stream pull request yesterday, not sure if you got notified.

**Comment by philipwinklerdelmic** [comment]
I added a new commit fixing the issue that we saw yesterday (no pixel duration when starting the time correlator).

**Pull request merged**